bazel/external/quiche.BUILD

```diff
@@ -369,7 +369,11 @@ envoy_cc_test_library(
 
 envoy_cc_test(
     name = "epoll_server_test",
-    srcs = ["quiche/epoll_server/simple_epoll_server_test.cc"],
+    srcs = select({
+        "@envoy//bazel:linux_x86_64": ["quiche/epoll_server/simple_epoll_server_test.cc"],
```

Comment:
I'm not super excited about encoded architecture (rather than OS) dependencies in various leaf locations. Can we have a `linux` config setting, similar to what is done for `apple` at https://github.com/envoyproxy/envoy/blob/master/bazel/BUILD#L236?

Reply:
Done.